src/ui/server/controllers/insurance/eligibility/total-value-insured/index.ts

```diff
@@ -36,20 +60,11 @@ export const post = (req: Request, res: Response) => {
       }),
       userName: getUserNameFromSession(req.session.user),
       validationErrors,
+      ...PAGE_VARIABLES,
     });
   }
 
-  const answer = req.body[FIELD_ID];
-
-  if (answer === 'true') {
-    const { INSURANCE } = PAGES;
-    const { APPLY_OFFLINE } = INSURANCE;
-    const { REASON } = APPLY_OFFLINE;
-
-    req.flash('exitReason', REASON.WANT_COVER_OVER_MAX_AMOUNT);
```

Comment:
Can `WANT_COVER_OVER_MAX_AMOUNT` be removed now?

Also It seems we need to update the XLSX, see `mapEligibility` in the API

Reply:
Bump `WANT_COVER_OVER_MAX_AMOUNT` be removed 🎉

Reply:
I'll do this in the next PR